third_party/compute_library/BUILD

```
@@ -29,6 +29,7 @@ _COMPUTE_LIBRARY_DEFINES = [
     "ENABLE_INTEGER_KERNELS",
     "ENABLE_NHWC_KERNELS",
     "ENABLE_NCHW_KERNELS",
+    "ARM_COMPUTE_ENABLE_FIXED_FORMAT_KERNELS"
```

The formatting presubmit complains about a missing comma at the end. Please fix.
```suggestion
    "ARM_COMPUTE_ENABLE_FIXED_FORMAT_KERNELS",
```

Error log:
```
diff --git a/third_party/compute_library/BUILD b/tmp/buildifier-tmp-292014704
index 86449751ace..68be8a22762 100644
--- a/third_party/compute_library/BUILD
+++ b/tmp/buildifier-tmp-292014704
@@ -29,7 +29,7 @@ _COMPUTE_LIBRARY_DEFINES = [
     "ENABLE_INTEGER_KERNELS",
     "ENABLE_NHWC_KERNELS",
     "ENABLE_NCHW_KERNELS",
-    "ARM_COMPUTE_ENABLE_FIXED_FORMAT_KERNELS"
+    "ARM_COMPUTE_ENABLE_FIXED_FORMAT_KERNELS",
 ]
```

---

Thanks @penpornk. Fixed with 803f6ed